zombpyg/game.py

```diff
@@ -252,9 +255,10 @@ def play_action(self, action_id, num_frames=1):
         rewards, observations, done, truncated = self.play_actions([action_id], num_frames=num_frames)
         return rewards[0], observations[0], done, truncated
     
-    # This is used by the train method
-    def get_total_reward(self):
-        return self.agent_rewards[0].get_total_reward()
+    # TODO
```

Comment:
Consider removing this method, or alternatively allowing for it to be called with an index specifying the agent.

Reply:
This will be removed.